docs/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>Clever Design System</title>
-    <link rel="shortcut icon" href="favicon.ico?1" >
+    <link rel="shortcut icon" href="favicon.ico" >
```

Comment:
I'm curious about this too! With the `?1`, when I replaced the favicon with the new one, it wouldn't reflect the changes.